hathor/conf/settings.py

```diff
@@ -154,6 +154,9 @@ def MAXIMUM_NUMBER_OF_HALVINGS(self) -> int:
     # that the peer is synced (in seconds).
     P2P_SYNC_THRESHOLD: int = 60
 
+    # Whether too warn the other peer of the reason for closing the connection
+    P2P_WARN_BLOCKED: bool = False
```

Comment:
This warning is only in the case of a whitelist rejection, right? Maybe we should make this explicit in the comment or in the var name

Reply:
What about `WHITELIST_WARN_BLOCKED_PEERS`?